.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Run Formater
```

Comment:
Can you change this file name to `lint.yml`? It'd make more sense as has no continuous integration related stuff on it.